state/event/event_test.go

```diff
@@ -29,7 +29,7 @@ func ExampleEvent() {
 	for _, ex := range examples {
 		fmt.Println("\nExample:", ex.name)
 
-		func() (string, error) {
+		_, _ = func() (string, error) {
```

Comment:
So these empty returns explicitly communicate that "I am consciously discarding the returns of this function" and I think that is valuable for readability.
Why this function even returns anything in the first place is another question I have, I was going to bring it up with Lorna after merging this PR.

Reply:
Yes, given that you're actively deciding not to use there error, then in this case I suppose it makes sense to include both blank identifiers. It would be good to understand more about why this function return `(string, error)`

Reply:
Per discussion with @wilkermichael, we will leave this as is, pending a conversation with @lornasong about the need for the return statements in the function. If they are not needed, they will be removed in a separate PR.

Reply:
Looks to me like the output of the anonymous functions aren't used. I can't remember what my intention was here - maybe it started off as test and then I converted to an example and didn't clean up? I'm ok with removing the return statements